supplementary_style_guide/glossary_terms_conventions/general_conventions/o.adoc

```diff
@@ -1,3 +1,75 @@
+// OCS: General; kept as is
+[discrete]
+[[object-bucket]]
+==== object bucket (noun)
+*Description*: A container to store objects.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+// OCS: General; kept as is
+[discrete]
+[[object-bucket-claim]]
+==== object bucket claim (noun)
+*Description*: Creates a new bucket and an application account in Multicloud Object Gateway or RADOS Gateway (RGW) with permissions to the bucket, including a new access key and secret access key. You can use object bucket claim to request an AWS S3 compatible bucket for the workloads.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*:
+
+// RHDS: General; kept as is
+[discrete]
+[[object-class]]
+==== image:images/yes.png[yes] object class (noun)
+*Description*: Object classes in an entry control which attributes are optional and which are required. Write as two words when you refer to object classes in general.
+
+*Use it*: yes
+
+*Incorrect forms*: objectClass
+
+*See also*: xref:objectclass[objectClass]
+
+// Ceph: Added "In Red Hat Ceph Storage, Object Storage Device is"
+[discrete]
+[[object-storage-device]]
+==== image:images/yes.png[yes] Object Storage Device (noun)
+*Description*: In Red Hat Ceph Storage, Object Storage Device is a storage drive in a Ceph Storage Cluster. Do not confuse Object Storage Device with the Ceph OSD, which is the `ceph-osd` daemon and the underlying data disk.
+
+*Use it*: yes
+
+*Incorrect forms*:
+
+*See also*: xref:ceph-osd[ceph-osd], xref:osd[OSD], xref:osd-daemon[OSD daemon]
+
+// Ceph: Added "In Red Hat Ceph Storage, Object Store is"
+[discrete]
+[[object-store]]
+==== image:images/yes.png[yes] Object Store (noun)
+*Description*: In Red Hat Ceph Storage, Object Store is a core component of the Ceph Storage Cluster. Also referred as RADOS.
```

Comment:
Typo: it should be "referred to as". This is not a full sentence either, so maybe add "It's" or "This component is" before "also referred to as RADOS".